lambda/update/main_test.go

```diff
@@ -296,7 +316,7 @@ func TestHandleEventWhenSendLpaUpdatedFailed(t *testing.T) {
 	}
 
 	resp, err := l.HandleEvent(context.Background(), events.APIGatewayProxyRequest{
-		Body: `{"type":"CERTIFICATE_PROVIDER_SIGN","changes":[{"key":"/certificateProvider/signedAt","old":null,"new":"2022-01-02T12:13:14.000000006Z"},{"key":"/certificateProvider/contactLanguagePreference","old":null,"new":"en"},{"key":"/certificateProvider/email","old":null,"new":"a@example.com"}]}`,
+		Body: `{"type":"CERTIFICATE_PROVIDER_SIGN","changes":[{"key":"/certificateProvider/signedAt","old":null,"new":"2022-01-02T12:13:14.000000006Z"},{"key":"/certificateProvider/contactLanguagePreference","old":null,"new":"en"},{"key":"/certificateProvider/email","old":null,"new":"b@example.com"},{"key":"/certificateProvider/channel","old":null,"new":"online"},{"key":"/certificateProvider/address/line1","old":null,"new":"Line 1"},{"key":"/certificateProvider/address/town","old":null,"new":"Town"},{"key":"/certificateProvider/address/country","old":null,"new":"GB"}]}`,
```

Comment:
I think this test was broken, as I had to add required fields to get what I assumed was the intended test scaffolding setup. Maybe? 🤷

Reply:
it passes on main though, so something has changed. I think those are all optional fields?

Reply:
Ah, I'd missed the high level address being optional and was focusing on the individual address parts. I'll fix up.

Reply:
can these be reverted now?